index.html

```diff
@@ -1,24 +1,137 @@
-<doctype html>
+<!doctype html>
 <html>
 
 <head>  
 <title> </title>
 <style type="text/css">
 	
 </style>
-
+<link type="text/css" rel="stylesheet" href="stylesheet.css" >
 </head>
 
 <body>
+<div>
+  <div id="navbar">
+	<div id="menu">
+		<button id="menu_button"><span><img src="images/menu-icon.png" ></span></button>
+	</div>
+	<div id="logo">
+		<a href="#"><img src="images/youtube-icon.png" height="25px" width="80px" ></a>
+	</div>
+	<div id="search-div">
+		<form>
+			<input type="text" name="" id="input1" placeholder="Search" />
+			<button name="" id="input2"><img src="images/search-icon.png" ></button>
+
+		</form>
+		
+	</div>
+	<div id="submenu">
+		<ul>
+			<li><a href="#"><img src="images/video-icon.jpg" /></a></li>
+			<li><a href="#"><img src="images/google-menu-icon.png" /></a></li>
+			<li><a href="#"><img src="images/notification-icon.jpg" /></a></li>
+			<li><a href="#"><img src="images/a-icon.jpg" id="a-icon" /></a></li>
+		</ul>
+	</div>
+		
+	
+  </div>
+
+
+
+
+  <div id="main_section">
+  	<div id="main">
+       <div id="" >
+       	 <iframe id="vid"  src="https://www.youtube.com/embed/xR-PjMUO9ds" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+       	 	
+       	 </iframe>
+        
+       </div>
+       <div style="">
```

Comment:
Let's remove all empty attributes. 
They are redundant and not need. 
Make sure you do this for all other occurrences.

Reply:
Thanks for your review. All empty attribute has been removed.